### PR TITLE
Internal: Added `@deprecated` tags to mark deprecated JS symbols [ED-10606]

### PR DESCRIPTION
**Pull request by rami-elementor**

### CODE REVIEW

File: assets/dev/js/editor/components/template-library/manager.js [hunk 1]
```diff
@@ -132,6 +132,9 @@ TemplateLibraryManager = function() {
 		dialog.show();
 	};
 
+	/**
```
Comment (udidol) [inline assets/dev/js/editor/components/template-library/manager.js:135]: See JS Lint

File: assets/dev/js/editor/container/container.js [hunk 2]
```diff
@@ -468,6 +468,9 @@ export default class Container extends ArgsObject {
 		return result;
 	}
 
+	/**
```
Comment (udidol) [inline assets/dev/js/editor/container/container.js:471]: See JS Lint

File: assets/dev/js/editor/container/container.js [hunk 3]
```diff
@@ -478,6 +481,9 @@ export default class Container extends ArgsObject {
 		return this.children.findRecursive( callback );
 	}
 
+	/**
```
Comment (udidol) [inline assets/dev/js/editor/container/container.js:484]: See JS Lint

**Review by udidol — CHANGES_REQUESTED** [review]

**Review by udidol — DISMISSED** [review]

File: assets/dev/js/editor/controls/repeater.js [hunk 4]
```diff
@@ -188,7 +188,12 @@ ControlRepeaterItemView = ControlBaseDataView.extend( {
 		this.updateActiveRow();
 	},
 
-	// BC since 3.0.0, ensure a new child is appear in container children.
+	/**
+	 * BC since 3.0.0, ensure a new child is appear in container children.
+	 *
+	 * @param {*} model - Container model.
+	 * @deprecated since 3.0.0, use `$e.run( 'document/repeater/insert' )` instead.
```
Comment (matipojo) [inline assets/dev/js/editor/controls/repeater.js:195]: It's not deprecated, only "add models directly to the repeater" is deprecated

File: assets/dev/js/frontend/utils/lightbox/lightbox.js [hunk 5]
```diff
@@ -238,9 +238,13 @@ module.exports = elementorModules.ViewModule.extend( {
 		this.openSlideshow( element.dataset.elementorLightboxSlideshow, initialSlideURL );
 	},
 
+	/**
+	 * @param {*} html
+	 * @deprecated since 3.1.4
```
Comment (matipojo) [inline assets/dev/js/frontend/utils/lightbox/lightbox.js:243]: Only `if ( window.elementorCommon )`

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by arielk — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**